src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -27,6 +31,17 @@ public class Hardware {
 	public static final Rotation2d DRIVEBASE_BACK_RIGHT_ENCODER_OFFSET =
 			Rotation2d.fromDegrees(311.396);
 	public static final int GYRO_PORT = 13;
+
+	// Cameras
+	public static final String PHOTON_CAM = "Integrated_Webcam";
+	// Rough measurements
+	public static final Transform3d ROBOT_TO_CAM =
+			new Transform3d(
+					new Translation3d(
+							Units.inchesToMeters(-8), Units.inchesToMeters(6), Units.inchesToMeters(30)),
```

Comment:
Might be good to use constants (especially if we are considering adding cameras to bonk for testing).